app/views/steps/client/contact_details/edit.html.erb

```diff
@@ -7,5 +7,10 @@
 
     <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
 
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_text_field :telephone_number, autocomplete: 'off', width: 'one-third', label: { size: 'm' } %>
```

Comment:
We can use `govuk_phone_field`, so on mobile devices the default keyboard is the numeric/phone one.

Reply:
Will add this